File: 151001/Timoshek/publisher/src/main/java/by/bsuir/egor/dto/StickerRequestTo.java
package by.bsuir.egor.dto;

public class StickerRequestTo {
    private long id;
    private String name;
    private  long issueId;

    public void setIssueId(long issueId) {
        this.issueId = issueId;
    }

    public long getIssueId() {
        return issueId;
    }

    public void setName(String name) {
        this.name = name;
    }

    public void setId(long id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public long getId() {
        return id;
    }
}

File: 151004/Korol/Lab5/publisher/DC_Lab1/DC_Lab1/DTO/StickerRequestTo.cs
using DC_Lab1.DTO.Interface;

namespace DC_Lab1.DTO
{
    public class StickerRequestTo(int? Id, string? Name) : IRequestTo
    {
        public int? Id { get; set; } = Id;

        public string? name { get; set; } = Name;
    }
}
